wbooks/src/app/screens/Library/index.tsx

```diff
@@ -28,12 +31,7 @@ function Library({ navigation }: Props) {
 
   return (
     <SafeAreaView style={styles.container}>
-      <FlatList
-        style={styles.container}
-        data={BOOKS_MOCK}
-        renderItem={renderBooks}
-        keyExtractor={keyExtractor}
-      />
+      <FlatList style={styles.container} data={books} renderItem={renderBooks} keyExtractor={keyExtractor} />
```

Comment:
Is it necessary to have the same style in the `SafeAreaView` and `FlatList`? I'm think not

Reply:
You were right, I removed it from the `FlatList` and there's no difference.